src/server/lib/logger.ts

```diff
@@ -1,34 +1,45 @@
 import { Logger, LogLevel } from '@/server/models/Logger';
 import { createLogger, transports } from 'winston';
+import * as util from 'util';
 
 const winstonLogger = createLogger({
   transports: [new transports.Console()],
 });
 
 export const logger: Logger = {
-  log(level: LogLevel, message: string) {
-    winstonLogger.log(level, message);
-  },
-  // errors can be anything
   // eslint-disable-next-line
-  error(message: string, error?: any) {
+  log(level: LogLevel, message: string, error?: any) {
```

Comment:
We seem to be assuming `message` exists on `error`. I know this is outside this PR but maybe we can type it as `unknown` and see what that throws up?

Reply:
I've made an improvement here to ensure error is a non null object before looking at its properties. Using `any` is necessary because JavaScript can throw `any`, and we want to be able to log all possible unexpected errors thrown by libraries which we don't control. https://github.com/guardian/gateway/pull/784/commits/381d8730143f13171dcf2b5011ef20c8ddde40c2